### PR TITLE
MAINT: bump napari-console to 0.0.6.

### PR DESCRIPTION
**Pull request by Carreau**
It does not exists yet, but will need to be published at some point.
In particular please see napari/napari-console#19 that tries to update
napari-console

Out of interest is bumping the minimum version of Python to 3.8,
and removal of deprecated napari API so that we can remove them from
napari.

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

### CODE REVIEW

**Comment by Carreau** [comment]
Note that this is mostly a placeholder to get review on napari/napari-console as otherwise it seem to fall through the cracks.

**Comment by alisterburt** [comment]
please ping me for merge here when napari-console 0.0.5 is released

We could try for 0.4.17 if you think 0.0.5 is good for release this week

**Comment by Carreau** [comment]
@sofroniewn it looks like you are the only maintainer for napari-console on PyPI, could you care to add a few other people from @napari/core-devs maybe ? 

And one of you will need to make a release (pushing a tag on eponymous repo should be enough).

**Comment by Carreau** [comment]
I've opened https://github.com/napari/napari-console/pull/20 as well to update the building process.

**Comment by andy-sweet** [comment]
@Carreau : I think @sofroniewn might not respond until next week, so if we want this in 0.4.17, we might need to wait until then (which should probably be fine).

File: setup.cfg [hunk 1]
```diff
@@ -50,7 +50,7 @@ install_requires =
     imageio>=2.5.0,!=2.11.0
     jsonschema>=3.2.0
     magicgui>=0.3.6
-    napari-console>=0.0.4
+    napari-console>=0.0.5
```
Comment (alisterburt) [inline setup.cfg]:
```suggestion
    napari-console>=0.0.6
```

**Review by alisterburt — APPROVED** [review]
0.0.6 released, merging here once CI passes!

**Pull request merged**